### PR TITLE
Add a simple README for Thanos

### PR DESCRIPTION
**Pull request by 4n4nd**

### CODE REVIEW

**Review by harshad16 — COMMENTED** [review]
/lgtm
:+1:

File: docs/observatorium/thanos/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+## Thanos Access
+
+[Thanos](https://github.com/thanos-io/thanos) is a set of components that can be composed into a highly available metric system with unlimited storage capacity, which can be added seamlessly on top of existing Prometheus deployments.
+
+An instance of Thanos is available for use in OperateFirst.
```
Comment (tumido) [inline docs/observatorium/thanos/README.md:5]:
```suggestion
An instance of Thanos is available for use in Operate First.
```

File: docs/observatorium/thanos/README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+Thanos access is open to public and the Query API/Console can be accessed at: <br>
```
Comment (tumido) [inline docs/observatorium/thanos/README.md:7]:
```suggestion
Thanos access is open to public and the Query API/Console can be accessed at:

```

File: docs/observatorium/thanos/README.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+In this instance of Thanos, metric data will be stored as following:
+
+| Resolution  | Duration    |
+| ----------- | ----------- |
+| Original    | 7 days      |
+| 5 min       | 30 days     |
+| 1 hour      | Indefinite  |
+
+*Note: the resolution and duration of the stored metric data might change in the future*
```
Comment (tumido) [inline docs/observatorium/thanos/README.md:19]: Can you link to the Thanos docs in here? Like what does the "original" resolution mean and so on.. Just some links explaining what the settings mean.

**Review by tumido — COMMENTED** [review]
/lgtm

**Review by tumido — APPROVED** [review]
/approve

**Pull request merged**